evalme/image/object_detection.py

```diff
@@ -527,17 +530,23 @@ def compare(self, pred,
             values = results.values()
             return sum(values) / len(values) if len(values) > 0 else None
 
-    def _get_max_iou_rectangles(self, gt, pred):
+    def _get_max_iou_rectangles(self, gt, pred, shape=None):
         """
         Get max iou for OCR shapes
         :param gt: Ground Truth result
         :param pred: Predicted result
         :return: Max score float[0..1]
         """
         max_score = 0
+        if shape == 'brushlabels':
+            iou = BrushEvalItem._iou
+        elif shape == 'polygonlabels':
+            iou = PolygonObjectDetectionEvalItem(raw_data=gt)._iou
+        else:
+            iou = self._iou
         for item in gt:
```

Comment:
Looks like it might be extremely slow if an annotation has a lot of brush regions.